bin/pocs

```diff
@@ -34,11 +40,24 @@ if [ "${START}" = 'help' ] || [ "${START}" = '-h' ] || [ "${START}" = '--help' ]
 	exit 1
 fi
 
+PARAMS="$@"   
+    
 cd "$PANDIR"
-docker-compose \
-    --project-directory "${PANDIR}" \
-	-f panoptes-utils/docker/docker-compose.yaml \
-	-f PAWS/docker/docker-compose.yaml \
-	-f POCS/docker/docker-compose.yaml \
-	-p panoptes "$@"
-
+CMD="docker-compose \
+    --project-directory ${PANDIR} \
+    -f panoptes-utils/docker/docker-compose.yaml \
+    -f PAWS/docker/docker-compose.yaml \
+    -f POCS/docker/docker-compose.yaml \
+    -p panoptes"
+    
+# If user only asked to start, check if already running and if so use "-d" option.
+if [[ "$PARAMS" == "up" ]]; then
+    if [[ ! -z $(eval "${CMD} top") ]]; then
```

Comment:
Why the change to `top` from `ps -q`? I think either will work, just curious.

Reply:
I replaced `ps -q` with `top` to filter for running processes. 

On our system, when we tried `bin/pocs up` with **no** containers running, the bash script added the `-d` option, which is not the intended behavior.  We echoed the parameter of the if statement (`$(eval "${CMD} ps -q")`), and saw that this parameter is the always the same, and does not change based on the status of the docker containers. My best guess as to why this happens is that `docker ps` lists running containers, while `docker-compose ps` lists any information it can get from the `docker-compose.yaml` files, which it can find when docker containers aren't running.

Using `top` lists only the running processes.

Reply:
Hm, that doesn't happen for me, `ps -q` will only give output. I guess maybe if you have some stopped machines? I tend to do `docker system prune` a lot as it cleans up a lot of hard drive space, but that would remove stopped and unused containers.